services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +43,47 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
+  if (!event) {
+    return ERR_CODE_INVALID_QUEUE_MSG;
+  } else if (!thermalMgrQueueHandle) {
+    return ERR_CODE_INVALID_STATE;
+  }
+  BaseType_t errCode = xQueueSend(thermalMgrQueueHandle, event, 0);
+  if (errCode == errQUEUE_FULL) {
```

Comment:
the error code variable is not necessary here, you can just do `if(xQueueSend(thermalMgrQueueHandle, event, 0) == errQUEUE_FULL)`